omnibus/config/templates/updater/datadog-agent-exp.service.erb

```diff
@@ -9,12 +9,13 @@ Wants=datadog-agent-trace-exp.service datadog-agent-process-exp.service datadog-
 
 [Service]
 Type=oneshot
-PIDFile=<%= install_dir %>/run/agent.pid
+PIDFile=<%= agent_dir %>/run/agent.pid
 User=dd-agent
 EnvironmentFile=-<%= etc_dir %>/environment
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= agent_dir %>/run/agent.pid
```

Comment:
As `agent_entrypoints` seems to have vanished everywhere else, is this one expected?

Reply:
nice catch, I missed changing this one